components/SideBar.vue

```diff
@@ -52,37 +58,45 @@
 <script lang="ts" setup>
 import { CIcon } from "@coreui/icons-vue";
 
+const { data: versionData } = useFetch("/api/versions");
+
+const versionTooltipContent = computed(() => {
+  if (versionData.value) {
+    return `Model version: ${versionData.value.daedalusModel} \nR API version: ${versionData.value.daedalusApi} \nWeb app version: ${versionData.value.daedalusWebApp}`;
+  } else {
+    return undefined;
+  }
+});
+
 const visible = defineModel("visible", { type: Boolean, required: true });
 const largeScreen = ref(true);
 
+const breakpoint = 992; // CoreUI's "lg" breakpoint
+const resetSidebarPerScreenSize = () => {
+  // Set the default values for the sidebar based on the screen size.
+  if (window.innerWidth < breakpoint) {
+    visible.value = false;
+    largeScreen.value = false;
+  } else {
+    visible.value = true;
+    largeScreen.value = true;
+  }
+};
+
 const hideHasNeverBeenEmitted = ref(true);
-function handleHide() {
+const handleHide = () => {
   if (hideHasNeverBeenEmitted.value) {
     // If this is the first 'hide', which is emitted on page load, we un-do it.
     // This is because the CoreUI Sidebar component emits a 'hide' event on page load, which we
     // don't want to obey for larger screen sizes.
     resetSidebarPerScreenSize();
     hideHasNeverBeenEmitted.value = false;
-  }
-  else {
+  } else {
     // If this is not the first 'hide', emitted on page load, we obey it and sync
     // the parent component's value.
     visible.value = false;
   }
-}
-
-const breakpoint = 992; // CoreUI's "lg" breakpoint
-function resetSidebarPerScreenSize() {
-  // Set the default values for the sidebar based on the screen size.
-  if (window.innerWidth < breakpoint) {
-    visible.value = false;
-    largeScreen.value = false;
-  }
-  else {
-    visible.value = true;
-    largeScreen.value = true;
-  }
-}
+};
```

Comment:
Sorry, should have commented to say this whole section is all formatting changes.